src/grandorgue/combinations/GOSetter.cpp

```diff
@@ -371,6 +401,13 @@ void GOSetter::Load(GOConfigReader &cfg) {
   m_buttons[ID_SETTER_INSERT]->Init(cfg, wxT("SetterInsert"), _("Insert"));
   m_buttons[ID_SETTER_DELETE]->Init(cfg, wxT("SetterDelete"), _("Delete"));
 
+  m_buttons[ID_SETTER_REFRESH]->Init(cfg, GROUP_REFRESH, _("Refresh"));
+  m_buttons[ID_SETTER_PREV_FILE]->Init(cfg, GROUP_PREV_FILE, _("Prev File"));
+  m_buttons[ID_SETTER_NEXT_FILE]->Init(cfg, GROUP_NEXT_FILE, _("Next File"));
+  m_buttons[ID_SETTER_LOAD_CMB]->Init(cfg, GROUP_LOAD_CMB, _("Load"));
```

Comment:
Load cmb

Reply:
I changed to "Load file" and "Save file". If you agree, give me a sign, and I will update the screenshot for help

Reply:
@oleg68 That's ok with me, I understand that it's difficult to fit the Load combinations and Save combinations. So if @rousseldenis doesn't have a better idea that fits, lets go with Load file and Save file, but maybe the Id constants should be changed too then.

Reply:
You are right. I replaced to FILE everywhere